create_db.sh

```diff
@@ -1,9 +1,11 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
Actually Galaxy starts fine for me on Ubuntu (which uses dash) when changing `s/bash/sh/` in `scripts/common_startup.sh` .